presidio-analyzer/presidio_analyzer/predefined_recognizers/in_passport_recognizer.py

```diff
@@ -21,7 +21,7 @@ class InPassportRecognizer(PatternRecognizer):
     PATTERNS = [
         Pattern(
             "PASSPORT",
-            r"\b[A-Z][1-9][0-9]{2}[0-9]{4}[1-9]\b",
+            r"\b[A-Z][1-9]\d\s?\d{4}[1-9]\b",
```

Comment:
It's hard to find a trustworthy reference for this. In some places it says that 0 is an option, except for the 2nd location.

Reply:
Found [this](https://www.geeksforgeeks.org/how-to-validate-indian-passport-number-using-regular-expression/) and [this](https://www.bajajallianz.com/blog/travel-insurance-articles/where-is-passport-number-in-indian-passport.html)

Reply:
Should the last [1-9] be [0-9]?

Reply:
Both references mention 1-9, where have you seen otherwise?

Reply:
oh I see. I got confused with the previous chars. Resolving